ng-dev/pr/merge/task.ts

```diff
@@ -167,4 +186,94 @@ export class PullRequestMergeTask {
       await strategy.cleanup(pullRequest);
     }
   }
+
+  /**
+   * Modifies the pull request in place with new target branches based on user selection from
+   * the available active branches.
+   */
+  private async setTargetedBranchesManually(pullRequest: PullRequest): Promise<void | MergeResult> {
+    const {mainBranchName, name, owner} = this.config.github;
+
+    // Attempt to retrieve the active LTS branches to be included in the selection.
+    let ltsBranches: {branchName: string; version: semver.SemVer}[] = [];
+    try {
+      assertValidReleaseConfig(this.config);
+      const ltsBranchesFromNpm = await fetchLongTermSupportBranchesFromNpm(this.config.release);
+      ltsBranches = ltsBranchesFromNpm.active.map(({name, version}) => ({
+        branchName: name,
+        version,
+      }));
+    } catch {
+      warn(
+        'Unable to determine the active LTS branches as a release config is not set for this repo.',
+      );
+    }
+
+    // Gather the current active release trains.
+    const {latest, next, releaseCandidate} = await fetchActiveReleaseTrains({
+      name,
+      nextBranchName: mainBranchName,
+      owner,
+      api: this.git.github,
+    });
+
+    // Collate the known active branches into a single list.
+    const activeBranches: {branchName: string; version: semver.SemVer}[] = [
+      next,
+      latest,
+      ...ltsBranches,
+    ];
+    if (releaseCandidate !== null) {
+      // Since the next version will always be the primary github branch rather than semver, the RC
+      // branch should be included as the second item in the list.
+      activeBranches.splice(1, 0, releaseCandidate);
+    }
+
+    const {selectedBranches, confirm} = await prompt([
+      {
+        type: 'checkbox',
+        default: pullRequest.targetBranches,
+        choices: activeBranches.map(({branchName, version}) => {
+          return {
+            checked: pullRequest.targetBranches.includes(branchName),
+            value: branchName,
+            short: branchName,
+            name: `${branchName} (${version}) ${
```

Comment:
real nit: the space should only be added when needed below

```suggestion
            name: `${branchName} (${version})${
```